main.py

```diff
@@ -361,19 +364,46 @@ def __init__(self, *args):
 		self.restartButton.clicked.connect(self.onRestartButtonClicked)
 		self.actionExit.triggered.connect(self.onExitClicked)
 		self.actionSettings.triggered.connect(self.onSettingsClicked)
+		self.inst._p.readyReadStandardOutput.connect(self.onReadyReadStandardOutput)
+		self.inst._p.readyReadStandardError.connect(self.onReadyReadStandardError)
+		self.inst._p.stateChanged.connect(self.onXqemuStateChanged)
+
+	def onReadyReadStandardOutput(self):
+		text = self.inst._p.readAllStandardOutput().data().decode()
+		self.log.moveCursor(QtGui.QTextCursor.End)
+		self.log.insertPlainText(text)
+		self.log.moveCursor(QtGui.QTextCursor.End)
+
+	def onReadyReadStandardError(self):
+		text = self.inst._p.readAllStandardError().data().decode()
+		self.log.moveCursor(QtGui.QTextCursor.End)
+		self.log.insertPlainText(text)
+		self.log.moveCursor(QtGui.QTextCursor.End)
```

Comment:
nit: Somehow this also doesn't scroll all the way to the bottom. There appear to be 2 blank lines at the end.